packages/integrations/cloudflare/README.md

```diff
@@ -68,17 +68,17 @@ $ pnpm install wrangler --save-dev
 
 It's then possible to update the preview script in your `package.json` to `"preview": "wrangler pages dev ./dist"`.This will allow you run your entire application locally with [Wrangler](https://github.com/cloudflare/wrangler2), which supports secrets, environment variables, KV namespaces, Durable Objects and [all other supported Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/#adding-bindings).
 
-## Access to the cloudflare runtime
+## Access to the Cloudflare runtime
 
-You have the posibility to access all the cloudflare bindings and environment variables from your astro pages and api routes through the adapter API
+You can access all the Cloudflare bindings and environment variables from your `.astro` pages and API routes through the adapter API.
```

Comment:
You can actually access them from any Astro components.

Reply:
```suggestion
You can access all the Cloudflare bindings and environment variables from Astro components and API routes through the adapter API.
```

Reply:
I figured! Perfect. Much better.